pkgs/by-name/go/google-alloydb-auth-proxy/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+buildGoModule rec {
+  pname = "google-alloydb-auth-proxy";
+  version = "1.11.3";
+
+  src = fetchFromGitHub {
+    owner = "GoogleCloudPlatform";
+    repo = "alloydb-auth-proxy";
+    rev = "v${version}";
```

Prefix tags with refs/tags/ to avoid conflict with branch names
```suggestion
    rev = "refs/tags/v${version}";
```